guides/common/assembly_importing-content.adoc

```diff
@@ -9,13 +9,79 @@ include::modules/proc_creating-a-custom-product.adoc[leveloffset=+1]
 include::modules/proc_adding-custom-rpm-repositories.adoc[leveloffset=+1]
 
 ifndef::satellite[]
-include::modules/proc_adding-custom-rpm-repositories-for-alma-linux-9.adoc[leveloffset=+1]
+:os_major: 9
+:gpg_url: http://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux-{os_major}
+:os_name: Alma Linux
```

Comment:
https://almalinux.org/ looks like it doesn't use a space:
```suggestion
:os_name: AlmaLinux
```

Consequently, I'd also use `AlmaLinux` instead of `Alma_Linux`.

This would be consistent with the applying errata file.

Reply:
Yes, applied to both occurrences.